src/components/invitation/InviteUserForm.tsx

```
@@ -37,7 +37,48 @@ export function InviteUserForm({ showTitle = true, onSuccess }: InviteUserFormPr
       const callbacks: InvitationServiceCallbacks = {
         onInvitationCreated: (invitation) => {
           console.log('Invitation created:', invitation);
-          // In a real app, you would send an email here
+          
+          // Track analytics event
+          try {
+            if (window.analytics) {
```

Potential Runtime Error: `window.analytics` might be undefined if the analytics script hasn't loaded, fails to load, is blocked by an ad blocker, or isn't included on the page. The `try...catch` block mitigates a crash, but the analytics event will be silently lost. Consider adding a check `if (typeof window.analytics !== 'undefined' && window.analytics)` for clarity or implementing a more robust analytics abstraction layer that handles initialization states.